packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/v3/aio/operations/_operation_group_two_operations.py

```diff
@@ -122,10 +122,15 @@ async def test_four(  # pylint: disable=inconsistent-return-statements
         content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
         cls: ClsType[None] = kwargs.pop("cls", None)
 
-        _json = None
-        _content = None
+        _json: Any = None
+        _content: Any = None
         if isinstance(input, (IO, bytes)):
             _content = input
+            if not content_type:
```

Comment:
this is a breaking change. we also explicitly typed content_type as optional for io bytes bc core can take care of it in this case, so we def should not raise

Reply:
The reason why we mark `content_type` as `optional` is that we can help users assign specific value according to input type. However, in this sceranio, All these 4 possible candidates`application/pdf', 'image/jpeg', 'image/png', 'image/tiff'` corresponds with `bytes` that we can never help users make right decision anymore. If we don't raise exception, it may bring potential risk for server.

Reply:
but there are content types that are optional, like `"appllication/octet-stream"`. we can't do this because it will be breaking

Reply:
updated